vms/client/nx_vms_client_core/src/utils/media/voice_spectrum_analyzer.cpp

```diff
@@ -88,21 +86,47 @@ void QnVoiceSpectrumAnalyzer::initialize(int srcSampleRate, int channels)
 }
 
 void QnVoiceSpectrumAnalyzer::processData(const qint16* sampleData, int sampleCount)
+{
+    processDataInternal(sampleData, sampleCount);
+}
+
+void QnVoiceSpectrumAnalyzer::processData(const qint32* sampleData, int sampleCount)
+{
+    processDataInternal(sampleData, sampleCount);
+}
+
+void QnVoiceSpectrumAnalyzer::processData(const nx::media::audio::Format& format, const void* sampleData, int sampleBytes) {
```

Comment:
Braces on the next line, and 100 symbols limit.
```suggestion
void QnVoiceSpectrumAnalyzer::processData(
    const nx::media::audio::Format& format,
    const void* sampleData,
    int sampleBytes)
{
```